integration/js/VideoTest.js

```diff
@@ -21,6 +22,9 @@ class VideoTest extends SdkBaseTest {
     await UserJoinedMeetingCheck.executeStep(this, session, this.attendeeId);
     await WaitForRemoteParticipantsToJoinMeeting.executeStep(this, session);
     await RosterCheck.executeStep(this, session, 2);
+    if (useVideoProcessor) {
```

Comment:
How is it different from the normal video test? We did not check here whether the filter is actually applied right?

Reply:
the step will click the filter button to apply processed stream and then as usual video test, remote video/local video check will take place.

Reply:
Yeah but the video check below only checks on whether there is a video played. Not if there is a filter applied. We need a ToDo to write a check for that.

Reply:
yeah, it's a TODO.